huaweicloud/services/workspace/resource_huaweicloud_workspace_desktop_pool_notification.go

```diff
@@ -0,0 +1,191 @@
+package workspace
+
+import (
+	"context"
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
```

Comment:
var desktopPoolNotificationNonUpdatableParams = []string{"pool_id", "notifications"}
```suggestion

var desktopPoolNotificationNonUpdatableParams = []string{"pool_id", "notifications"}

```

For the ForceNew fields, except for region, you need to add CustomizeDiff and add NonUpdatable to the corresponding fields in the document.

Reply:
done